docs/install.md

```diff
@@ -332,6 +332,9 @@ for a complete list of supported properties.
 
 Here is a minimal example driver deployment created with a custom resource:
 
+**NOTE**: `nodeSelector` must match with node's label set in
+[Installation and setup](#installation-and-setup) section.
```

Comment:
I think its better to say "must match the node label that was set in the [Installation and setup](#installation-and-setup) section.".

"label set" could also be a set of labels, and the definite article clarifies that this sentence is talking about something specific.